src/path.js

```diff
@@ -23,12 +23,33 @@ function appendRound(digits) {
   };
 }
 
+function round(digits) {
+  const k = 10 ** digits;
+  return function(value) {
+    return Math.round(value * k) / k;
+  };
+}
+
+function equal(x0, y0, x1, y1) {
+  return x0 === x1 && y0 === y1;
+}
+
+function equalRound(digits) {
+  let d = Math.floor(digits);
+  if (d > 15) return equal;
+  const r = round(digits);
+  return function(x0, y0, x1, y1) {
+    return r(x0) === r(x1) && r(y0) === r(y1);
+  };
+}
+
```

Comment:
```suggestion
function equal(x0, y0, x1, y1) {
  return x0 === x1 && y0 === y1;
}

function equalRound(digits) {
  const d = Math.floor(digits);
  if (d > 15) return equal;
  const k = 10 ** d;
  const r = function(value) {
    return Math.round(value * k);
  };
  return function(x0, y0, x1, y1) {
    return r(x0) === r(x1) && r(y0) === r(y1);
  };
}
```

Skip the division, that's 5% ~ 10% faster. Use `d` rather than `digits` to compute the power of 10.

Note: the "round" function does not round anymore, but we inline it so we don't have to think of a different name.